src/store/model/laptopsdirect-ie.ts

```diff
@@ -0,0 +1,24 @@
+import {Store} from './store';
+
+export const LaptopsdirectIE: Store = {
+  currency: '€',
+  labels: {
+    inStock: {
+      container: 'span.atb',
+      text: ['Add to Basket »'],
```

Comment:
```suggestion
      text: ['Add to Basket'],
```

This should be sufficient